plugins/powerd/meson.build

```diff
@@ -0,0 +1,26 @@
+if host_machine.system() == 'linux'
+  cargs = ['-DG_LOG_DOMAIN="FuPluginPowerd"']
+
+shared_module('fu_plugin_cros_powerd',
```

Comment:
i think either the module needs to be called fu_plugin_powerd or the directory needs to be cros-powerd. I guess it depends on whether powerd is CrOS specific or not.

Reply:
That makes sense. I think we've settled on just fu-plugin-powerd.